### ISSUE TITLE
Migration of CFMS Discounts

### ISSUE THREAD

**Issue by hyan2017** [cfms-website/cfms-website.github.io#2]
Migration of the CFMS Discounts (under the benefits tab of http://www.old.cfms.org/) to the new website. Kevan Lu has already created the page for travel discounts (https://www.cfms.org/resources/discounts-travel.html), however the other discount categories need to be migrated. 

Specifically, UpToDate requests that their page be moved over by SGM as they are hoping to include promotional items in our Matchbook, including the online page hosting their discount.

**Comment by erm619** [comment]
I would also like to change the format on the new website so that the content can be updated and translated, instead of being on a photo form.

**Comment by pyarmak** [comment]
@VP-Communications as you can see from the commits above, this issue ended up being far more involved than the estimate would suggest.
Justin made the discounts far more user-friendly to manage and set it up in such a way that we can integrate my fix to the security concern far more easily.

**Comment by Threatsignal** [comment]
@VP-Communications Just submitted a pull request for this issue see #77 .

This issue is almost finished. Just need someone to translate the pages for the discounts section. I translated what I could with what I had available on the old.cfms website, however, there was a lot of information missing for the french section.

To edit this content it will now be available for editing in the discounts.yml file, when this pull request is accepted it will be available via cloud cannon.

Seperating all of the data from the view will also allow Pavel to Integrate his security fix into this discounts section. I had communicated with him and tested it as I reworked this section of the website.
